src/utils/getDistanceBetweenHTMLElements.ts

```diff
@@ -0,0 +1,22 @@
+export function getOffset(element: Element): { top: number; left: number; } | null {
+  if (!element.getClientRects().length) {
+    return null;
+  }
+
+  const rect = element.getBoundingClientRect();
+  const win = element.ownerDocument.defaultView;
+  if (!win) return null;
+  return {
+    top: rect.top + win?.scrollY,
+    left: rect.left + win?.scrollY,
+  };
```

Comment:
I believe we can just use the `window` global
```suggestion
  return {
    top: rect.top + window.scrollY,
    left: rect.left + window.scrollY,
  };
```